### PR TITLE
update the date time in example requests for customerSpecifiedDeliveryDateTime

### PR DESCRIPTION
**Pull request by AnshikaKoul**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/mrb13Jfa](https://trello.com/c/mrb13Jfa)

File: content/api/booking/examples/basicRequestNew.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "schemaVersion": 1,
   "consignments": [
     {
-      "shippingDateTime": 1588678607000,
-      "customerSpecifiedDeliveryDateTime": "2020-05-11",
+      "shippingDateTime": "2021-03-05T10:00:00",
```
Comment (tomgag) [inline content/api/booking/examples/basicRequestNew.json:6]: 👍

**Review by tomgag — APPROVED** [review]
LGTM 👍

File: content/api/booking/examples/basicRequestNew.json (hunk 1 above)
Comment (AnshikaKoul) [reply content/api/booking/examples/basicRequestNew.json:6]: Screenshot 2021-03-16 at 10 10 53 AM

Comment (AnshikaKoul) [reply content/api/booking/examples/basicRequestNew.json:6]: yes it does. but i realised in the example i have bungled up the year, will fix that

**Pull request merged**